src/envs/cover.py

```diff
@@ -779,7 +783,7 @@ def _Pick_learned_equivalent_policy(
         y = s.get(self._robot, "y")
         by = s.get(obj, "y")
         desired_x = p[7]
-        desired_y = by
+        desired_y = by + 1e-3
```

Comment:
@ashayathalye note: I added this because I saw learned policies colliding with the blocks they were trying to pick very frequently; this seemed to help

Reply:
I realized belatedly that I did not update the sampler accordingly 🤦 and your check in the terminal didn't catch this because the tolerance is 1e-2 and this was 1e-3. actually, I want to revisit why the tolerance is so high, that seems potentially problematic...

Reply:
I fixed the other one -- see the change in the oracle place sampler, and the reduction in tolerances in the terminal functions.